packages/repack/android/src/main/java/com/callstack/repack/ChunkManagerPackage.kt

```diff
@@ -1,17 +1,35 @@
 package com.callstack.repack
 
-import com.facebook.react.ReactPackage
-import com.facebook.react.bridge.NativeModule
+import com.facebook.react.TurboReactPackage
 import com.facebook.react.bridge.ReactApplicationContext
-import com.facebook.react.uimanager.ViewManager
-
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.module.model.ReactModuleInfoProvider
+import com.facebook.react.module.model.ReactModuleInfo
+import java.util.HashMap
 
-class ScriptManagerPackage : ReactPackage {
-    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
-        return listOf(ScriptManagerModule(reactContext))
+class ScriptManagerPackage : TurboReactPackage() {
+    override fun getModule(name: String, reactContext: ReactApplicationContext): NativeModule? {
+        return if (name == ScriptManagerModule.NAME) {
+            ScriptManagerModule(reactContext)
+        } else {
+            null
+        }
```

Comment:
nit: Not needed `else {}`

Reply:
<img width="840" alt="Screenshot 2024-02-01 at 10 02 04" src="https://github.com/callstack/repack/assets/12766071/9e37e2ff-e46b-4b0c-998b-8067b08082e1">

It seems to be needed :D